ledger-core/v2/logicrunner/builtin/builtin.go

```diff
@@ -121,5 +121,5 @@ func (b *BuiltIn) CallMethod(ctx context.Context, callCtx *insolar.LogicCallCont
 		return nil, nil, errors.New("failed to find contracts method")
 	}
 
-	return methodFunc(data, args)
+	return methodFunc.Func(data, args)
```

Comment:
maybe rename to methodObject?

Reply:
Old codebase, just supported compilation

Reply:
https://insolar.atlassian.net/browse/PLAT-298